mod_test.ts

```diff
@@ -949,25 +949,91 @@ const TXT_PLUGIN: esbuildNative.Plugin = {
   },
 };
 
-Deno.test("txt plugin", async (t) => {
+Deno.test("css plugin, entrypoint", async (t) => {
   +await testLoader(t, LOADERS, async (esbuild, loader) => {
     const res = await esbuild.build({
       ...DEFAULT_OPTS,
       plugins: [
-        denoResolverPlugin(),
-        TXT_PLUGIN,
-        denoLoaderPlugin({ loader }),
+        ...denoPlugins({ loader }),
+        CSS_PLUGIN,
```

Comment:
The CSS_PLUGIN in these tests can be after both deno plugins because it doesn't have an onResolve function that needs to be called. Unlike this fake one, my postCSSPlugin that the issue was originally created for does have an onResolve function. For it to work, it needs to be somewhere before the denoLoaderPlugin since denoLoaderPlugin's onResolve will catch all still.

For my postCSSPlugin to use the resolution defined in the denoResolverPlugin, it needs to be put between denoResolverPlugin and the denoLoaderPlugin.

Reply:
If my postCSSPlugin is put after the denoLoaderPlugin, it's onResolve will never get called, resulting in esbuild using the default css loader instead of my postcss loader.

I think it's fine to say that it may be desirable to put some plugins between denoResolvePlugin and denoLoaderPlugin. Then if your plugin doesn't need deno's resolution, it can just be placed before denoPlugins.